app/presenters/v3/process_presenter.rb

```diff
@@ -18,6 +18,9 @@ def associated_resources
         def to_hash
           health_check_data = { timeout: process.health_check_timeout, invocation_timeout: process.health_check_invocation_timeout }
           health_check_data[:endpoint] = process.health_check_http_endpoint if process.health_check_type == HealthCheckTypes::HTTP
+
+          readiness_health_check_data = { invocation_timeout: process.readiness_health_check_invocation_timeout }
```

Comment:
@sethboyles `health-check-timeout` represents start up timeout. When app starts up we run liveness health check for this period to mark it as Running. If it never succeeds for the first time during that timeout period we mark app as crashed and we don't event restart it since it was never running. Readiness health checks don't have that behavior. They start running after app instance is started and marked as Running.